### DISCUSSION TITLE
Zoomed in, broken lines / missing data

### DISCUSSION THREAD

**Discussion by Nick-Holaspa** [predict-idlab/plotly-resampler#255] [General]
Hi,

Setup is very similar to that shown [here](https://github.com/predict-idlab/plotly-resampler/blob/main/examples/dash_apps/02_minimal_cache.py). Perhaps the only difference is:

```
figure = FigureResampler(
    px.line(
        df,
         x = "x",
         y = "y",
         color = "color",
         line_shape = "hv"))

return figure, Serverside(figure)
```

At full zoom, the "[R]" is not displayed in the legend, but data does appear to be missing:

![Capture](https://github.com/predict-idlab/plotly-resampler/assets/36716609/50acda4a-375d-4d78-9953-1bd6be2e06b2)

Assuming the data is stored "in full" on the server, I'm not sure why there appears to be loss?

Appreciate any advice.

Kind regards,

Nick.

**Comment by jonasvdd** [comment]
Hi @Nick-Holaspa,

As this discussion is closed, I presume that you found a working solution? 

Kind regards,
Jonas
